docs/_docs/collections.md

```diff
@@ -49,10 +49,10 @@ defaults:
 <div class="note">
   <h5>Gather your collections {%- include docs_version_badge.html version="3.7.0" -%}</h5>
 
-  <p>You can optionally specify a directory to store all your collections in the same place with <code>collections_dir: my_collections</code></p>
+  <p>You can optionally specify a directory to store all your collections in the same place with <code>collections_dir: my_collections</code>.</p>
 
   <p>Then Jekyll will look in <code>my_collections/_books</code> for the <code>books</code> collection, and
-  in <code>my_collections/_recipes</code> for the <code>recipes</code> collection.</p>
+  in <code>my_collections/_recipes</code> for the <code>recipes</code> collection. Since posts and drafts are collections as well, you will need to move the <code>_drafts</code> and <code>_posts</code> directories into <code>my_collections</code> as well.</p>
```

Comment:
Should we turn this into a warning?

Reply:
drafts are not a collection! They're just a temporary holding..

Reply:
I've updated my previous PR to confirm that drafts at the root are processed even when a custom collections_dir is set..

Reply:
@ashmaroli 

How about the issue, though, that when you have collections in a custom collections directory and posts at the root, that the posts will have a draft status (regardless of whether publishing date is in the past or not)?